Milestone 7/script.js

```diff
@@ -0,0 +1,286 @@
+// UI Elements 
+const isButton = document.querySelector('.is-button')
+const result = document.querySelector('.result')
+const userInput = document.querySelector('.user-input')
+const spinnerElement = document.querySelector('#spinnerElement')
+const errorMessage = document.querySelector('.error-message')
+const resultsContainer = document.querySelector('.results-container')
+const checkBox = document.querySelector('#check-box')
+const form = document.querySelector('.form')
+const select = document.querySelector('select')
+
+
+// Hide Error on Default 
+hideError()
+
+
+// Event Listerners
+
+
+// Hide Spinner On Default
+hideSpinner()
+
+
+
+// Fucntions    // Calc Fibo Main Fucntion
+
+
+function calcFibo(e) {
+
+    if (userInput.value === '') return
+    userInput.style.borderColor = 'black'
+    hideError()
+
+    const number = userInput.value
+
+    if (number > 50) {
+        showError()
+        errorMessage.innerHTML = ''
+        return
+    }
+
+    showSpinner()
+
+
+
+
+    // Fetch data from local Server
+    fetch(`http://localhost:5050/fibonacci/${number}`)
+
+
+        .then(data => {
+            if (!data.ok) {
+                data.text().then(data => {
+                    result.innerHTML = `error : ${data}`
+                    presentError42()
+                    throw new Error(data)
```

Comment:
You are throwing the error and the .catch will catch it, so you should display the error in the .catch instead of here